cpu/kinetis/include/bme.h

```diff
@@ -246,6 +247,63 @@ static inline void bit_clear8(volatile uint8_t *ptr, uint8_t bit)
     *((volatile uint8_t *)(((uintptr_t)ptr) | BME_AND_MASK)) = (uint8_t)(~(1ul << bit));
 }
 
+/**
+ * @brief Checks if a single bit in the 32 bit word pointed to by @p ptr is set.
+ *
+ * The effect is the same as for the following snippet:
+ *
+ * @code{c}
+ *   *ptr & (1 << bit);
+ * @endcode
+ *
+ * @param[in] ptr   Pointer to target word.
+ * @param[in] bit   Bit number within the word.
+ *
+ * @return          True if the bit was set, false otherwise.
+ */
+static inline bool bit_check32(volatile uint32_t *ptr, uint8_t bit)
+{
+    return *((volatile uint32_t *)(((uintptr_t)ptr) | BME_AND_MASK)) & (uint32_t)(1ul << bit);
```

Comment:
Wouldn't this be
```suggestion
    return bme_bitfield32(ptr, bit, 32);
```

I have `openlabs-kw41z-mini` at home and can test

Reply:
Friendly ping @benpicco :)

Reply:
Sorry, I didn't get to it before my vaccation - I added a unit test so we can actually test those function and turns out `bit_clearXX()` is already not working :confused: 

```
2025-09-30 00:44:56,723 # START
2025-09-30 00:44:56,723 # .clear bit 0
2025-09-30 00:44:56,723 # clear bit 2
2025-09-30 00:44:56,723 # 
2025-09-30 00:44:56,724 # bit_tests.test_bit8 (tests/unittests/tests-bit/tests-bit.c 36) exp 250 was 251
2025-09-30 00:44:56,724 # .
2025-09-30 00:44:56,735 # bit_tests.test_bit16 (tests/unittests/tests-bit/tests-bit.c 60) exp 65530 was 65531
2025-09-30 00:44:56,735 # .
2025-09-30 00:44:56,740 # bit_tests.test_bit32 (tests/unittests/tests-bit/tests-bit.c 84) exp 4294967290 was 4294967291
```